package.json

```
@@ -8,11 +8,13 @@
     "build": "ng build --configuration production",
     "build:dev": "ng build --configuration development",
     "test": "ng test --watch=false",
-    "lint": "npx eslint 'src/**/*.{js,ts}' --quiet --fix",
+    "lint": "yarn eslint 'src/**/*.{js,ts,html}' --fix",
```

You shouldn't need the `yarn` prefix either.

```suggestion
    "lint": "eslint 'src/**/*.{js,ts,html}' --fix",
```